betty/test_utils/json/schema.py

```diff
@@ -39,40 +43,53 @@ class SchemaTestBase:
     A base class for testing :py:class:`betty.json.schema.Schema` implementations.
     """
 
-    async def get_sut_instances(self) -> Sequence[tuple[Schema, Sequence[Dump]]]:
+    @staticmethod
```

Comment:
Make this an instance method again. Not that it needs to be either kind of method, but it's ever so slightly more minimal than a class method.